scripts/lochness_create_template.py

```diff
@@ -233,26 +233,31 @@ def create_config_template(config_loc: Path, args: object) -> None:
         for study in args.studies:
             line_to_add = f'''
     {study}:
-        namespace: /DATA/ROOT/UNDER/BOX
+        namespace: /DATA/ROOT/UNDER/MEDIAFLUX
         delete_on_success: False
         file_patterns:
             actigraphy:
                 - vendor: Philips
                   product: Actiwatch 2
-                  data_dir: all_BWH_actigraphy
-                  pattern: 'accel/*csv'
+                  data_dir: actigraphy
```

Comment:
Just so we are on the same page--Justin showed the remote name to be more like the long name I have written: `all_BWH_actigraphy`. Your suggested change means a user would have to change the proposed `actigraphy` name to the actual remote name.

Reply:
Yes, correct. This is the new structure we already shared to the Prescient and Pronet side, and they will try to match it to this example.